dbfit-java/sqlserver/src/main/java/dbfit/environment/SqlServerEnvironment.java

```diff
@@ -67,9 +68,16 @@ protected String parseCommandText(String commandText) {
 
     public Map<String, DbParameterAccessor> getAllColumns(String tableOrViewName)
             throws SQLException {
+
+        String objectDatabaseName = "";
+        String[] objnameParts = tableOrViewName.split("\\.");
+        if (objnameParts.length == 3) {
+            objectDatabaseName = objnameParts[0] + ".";
```

Comment:
It sounds a bit odd at first glimpse to have something named `objectDatabaseName` to end with `.` (I wonder if we can figure out a better name).